.chloggen/26304-checkapi-nsxtreceiver.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
since this is test code, it's not considered a breaking change, and you can delete the changelog entry. I'll mark this PR as "Skip Changelog". Changes look good otherwise.

Reply:
Ah good point! Thank you! Will do

Reply:
is this `enhancement` or `bug_fix` or does it even matter?

Reply:
You can simply remove this file since it's a test-only change.

Reply:
oh gotcha. Yeah that was in the sentence sure enough... Some days... :brain: